RZViewActions/UIView+RZViewActions.m

```diff
@@ -90,9 +97,16 @@ + (instancetype)action:(RZViewActionBlock)action withDuration:(NSTimeInterval)du
 
 + (instancetype)action:(RZViewActionBlock)action withOptions:(UIViewAnimationOptions)options duration:(NSTimeInterval)duration
 {
-    NSParameterAssert(action);
+    return [[RZViewAction alloc] initWithBlock:action options:options duration:duration];
```

Comment:
I'm going to leave it as `[RZViewAction alloc]` and change all the `instancetype`s to `RZViewAction *` since that is more correct.